### ISSUE TITLE
Crash on AwaitView when DNKA is active 

### ISSUE THREAD

**Issue by ruimendesM** [Adyen/adyen-android#1234]
**Describe the bug**
Application Crashes when trying to pay with MBWay with DNKA (Do Not Keep Activities) active, if the user goes to background and then to foreground again.

This is problematic because MBWay will force the user to navigate to a different app to confirm the payment, which can increase the probability of this issue to occur in production.

**To Reproduce**
1. Make sure the device is with DNKA active
2. Start a payment with MBWay
3. While the payment is awaiting confirmation (see screenshot) go to background
4. Bring the app again back to foreground
5. Crash happens

**Expected behavior**
When returning to the app, the DropIn would be restored with the AwaitView and the process would conclude as expected.

**Screenshots**
<img src="https://github.com/Adyen/adyen-android/assets/19568050/84de811b-9120-4524-9339-07131048e6a2" width="200" height="400" />

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Any
 - Version: 4.12.0

**Additional context**
Stacktrace:
```
java.lang.NullPointerException:
Attempt to invoke virtual method 'int java.lang.String.hashCode()' on a null object reference
                  at com.adyen.checkout.await.AwaitView.getMessageTextResource(AwaitView.java:126)
                  at com.adyen.checkout.await.AwaitView.updateMessageText(AwaitView.java:118)
                  at com.adyen.checkout.await.AwaitView.onChanged(AwaitView.java:95)
                  at com.adyen.checkout.await.AwaitView.onChanged(AwaitView.java:32)
                  at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
                  at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:146)
                  at androidx.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:468)
                  at androidx.lifecycle.LiveData$LifecycleBoundObserver.onStateChanged(LiveData.java:425)
                  at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:360)
                  at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.java:271)
                  at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:313)
                  at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:151)
                  at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:134)
                  at androidx.fragment.app.FragmentViewLifecycleOwner.handleLifecycleEvent(FragmentViewLifecycleOwner.java:93)
                  at androidx.fragment.app.Fragment.performStart(Fragment.java:3169)
                  at androidx.fragment.app.FragmentStateManager.start(FragmentStateManager.java:588)
                  at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:279)
                  at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
                  at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1433)
                  at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2977)
                  at androidx.fragment.app.FragmentManager.dispatchStart(FragmentManager.java:2902)
                  at androidx.fragment.app.FragmentController.dispatchStart(FragmentController.java:274)
                  at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:359)
                  at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:248)
                  at com.adyen.checkout.dropin.ui.DropInActivity.onStart(DropInActivity.kt:248)
                  at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1455)
                  at android.app.Activity.performStart(Activity.java:8076)
                  at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3660)
                  at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
                  at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
                  at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
                  at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
                  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210)
                  at android.os.Handler.dispatchMessage(Handler.java:106)
                  at android.os.Looper.loopOnce(Looper.java:201)
                  at android.os.Looper.loop(Looper.java:288)
                  at android.app.ActivityThread.main(ActivityThread.java:7839)
                  at java.lang.reflect.Method.invoke(Native Method)
                  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```

**Comment by OscarSpruit** [comment]
Hi @ruimendesM, thank you for reporting this issue! I was able to reproduce it and we will look into a fix. I'll keep this issue posted

**Comment by jreij** [comment]
Hi, we just released [4.13.0](https://github.com/Adyen/adyen-android/releases/tag/4.13.0) in which this crash should be fixed.
